accessibility_monitoring_platform/apps/reports/views.py

```diff
@@ -37,12 +37,7 @@ def create_report(request: HttpRequest, case_id: int) -> HttpResponse:
         return redirect(
             reverse("cases:edit-report-ready-for-qa", kwargs={"pk": case.id})
         )
-    if case.audit is not None and not case.audit.uses_statement_checks:
-        report: Report = Report.objects.create(
-            case=case, report_version="v1_1_0__20230421"
```

Comment:
Why is the version removed?

Reply:
That is the last version of the report which showed the old statement content checking.

All (non-archived) Cases now use the new Yes/No statement content checking process.

So no more of these old-style reports can be created.

This entire `if` statement is replaced by an unconditional report create statement just below in the source code.